### PR TITLE
Fixed bounding box center calculation

### LINKED ISSUE #45: Tracker not keeping center of bounding box in frame
The AI tracker is not keeping the center of the bounding box inside of the "acceptable box". The "center" of the bounding box is currently being calculated to be on the side of one of the boxes. Fix this so that the center of the bounding box is always kept inside of the acceptable box. I believe the error is likely in `calculate_center_bounding_box()` in basic_directory.py.

### PR DESCRIPTION
**Pull request by nolanporter77**
closes #45

### CODE REVIEW

File: directors/basic_director.py [hunk 1]
```diff
@@ -109,7 +110,7 @@ def process_frame(self, frame : list):
                             print(rotation)
                             rotation = int(round(rotation))
                             #Publisher.rotate_altitude(rotation)
-                            Publisher.polar_pan(0, rotation, 0, 3000)
+                            #Publisher.polar_pan_discrete(0, rotation, 0, 3000)
```
Comment (DevanTurtle7) [inline directors/basic_director.py:113]: Should this be commented out?

Comment (nolanporter77) [reply directors/basic_director.py:113]: So when we were testing on Saturday it was commented out. I wasn't sure if it being commented out was part of the demo

Comment (nolanporter77) [reply directors/basic_director.py:113]: Just for the altitude not the azimuth

**Review by DevanTurtle7 — APPROVED** [review]

**Comment by nolanporter77** [comment]
PR Confidence: 5
Bug fixed

**Pull request merged**